acceptance/acceptance_test.go

```diff
@@ -30,11 +31,16 @@ var KeepTmp bool
 // example: var SingleTest = "bundle/variables/empty"
 var SingleTest = ""
 
+// TODO: Add a custom diff function based on the method and path.
+
 // If enabled, instead of compiling and running CLI externally, we'll start in-process server that accepts and runs
 // CLI commands. The $CLI in test scripts is a helper that just forwards command-line arguments to this server (see bin/callserver.py).
 // Also disables parallelism in tests.
 var InprocessMode bool
 
+// TODO: Acceptance tests are taking long to run. Is it possible to shorten
```

Comment:
Please clean up, this is not the right place. I started a Slack thread on slow tests.